lib/node_modules/@stdlib/math/base/special/expm1rel/lib/main.js

```diff
@@ -26,10 +38,16 @@
 *
 * @example
 * var v = expm1rel( 0.0 );
-* // returns TODO
+* // returns 1.0
```

Comment:
Same comment as above. We can include `1.0` and `-1.0` as example invocations here. Meaning, feel free to add a couple more examples.